libs/ui-lib/lib/common/config/constants.ts

```diff
@@ -16,7 +16,7 @@ export const getShortOpenshiftVersion = (ocpVersion?: string) =>
   ocpVersion?.split('.').slice(0, 2).join('.');
 
 export const ASSISTED_INSTALLER_DOCUMENTATION_LINK =
-  'https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2022/html/assisted_installer_for_openshift_container_platform/index';
+  'https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2024/html/installing_openshift_container_platform_with_the_assisted_installer/index';
```

Comment:
Hard-coded version numbers are problematic. It is better to link to the splash page "https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform", which resolves to the latest version.